### PR TITLE
[chrome] add printing namespace

### PR DESCRIPTION
**Pull request by erwanjugand**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developer.chrome.com/docs/extensions/reference/api/printing
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

A screenshot of runtime : 
image

**Notes** 

The documentation shows contradictory information for `submitJob`. The function description indicates `Before Chrome 120, this function did not return a promise`, but the Promise indicates `Chrome 100+`.

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -6742,6 +6742,159 @@ declare namespace chrome.printerProvider {
...
+         * The print job to be submitted.
+         * The only supported content type is "application/pdf", and the Cloud Job Ticket shouldn't include FitToPageTicketItem, PageRangeTicketItem, ReverseOrderTicketItem and VendorTicketItem fields since they are irrelevant for native printing.
+         * All other fields must be present.
+         */
+        job: chrome.printerProvider.PrintJob;
+    }
+
+    export interface SubmitJobResponse {
+        /** The id of created print job. This is a unique identifier among all print jobs on the device. If status is not OK, jobId will be null. */
+        jobId?: string;
```
Comment (nicolas377) [inline types/chrome/index.d.ts:6845]: If it's specifically string | null, go ahead and explicitly declare that. Declaring with `TypeName?: T` is a shortcut for `TypeName: T | undefined`, so you'll get an unexpected type here.

File: types/chrome/index.d.ts [hunk 2]
```diff
@@ -6742,6 +6742,159 @@ declare namespace chrome.printerProvider {
...
+ * Use the chrome.printing API to send print jobs to printers installed on Chromebook.
+
+ * Permissions: "printing"
+ * @platform ChromeOS only
+ * @since Chrome 81
+ */
+declare namespace chrome.printing {
+    export interface GetPrinterInfoResponse {
+        /** Printer capabilities in [CDD format](https://developers.google.com/cloud-print/docs/cdd#cdd-example). The property may be missing. */
+        capabilities?: object;
```
Comment (nicolas377) [inline types/chrome/index.d.ts:6758]: The type `object` is generally discouraged. The type `object` represents any non-primitive type, which is way too broad. Looks like you're trying to make a type for a generic object, so I'd suggest using `{ [key: string]: unknown }`.

**Review by nicolas377 — CHANGES_REQUESTED** [review]

File: types/chrome/index.d.ts (hunk 2 above)
Comment (erwanjugand) [reply types/chrome/index.d.ts:6758]: I was hesitant to strictly follow the doc by putting `object` but I prefer your suggestion :)

File: types/chrome/index.d.ts (hunk 1 above)
Comment (erwanjugand) [reply types/chrome/index.d.ts:6845]: The documentation misled it with the `optional` tag

After verification, the suggested type is correct :)

**Comment by erwanjugand** [comment]
Hey @nicolas377, just a friendly reminder to review the code when you have a chance 😇

**Review by nicolas377 — APPROVED** [review]

**Pull request merged**